zk-sdk/src/elgamal_program/errors.rs

```diff
@@ -1,7 +1,54 @@
-use thiserror::Error;
+use {
+    crate::{
+        errors::ElGamalError,
+        range_proof::errors::{RangeProofGenerationError, RangeProofVerificationError},
+        sigma_proofs::errors::*,
+    },
+    thiserror::Error,
+};
+
+#[cfg(not(target_os = "solana"))]
+#[derive(Error, Clone, Debug, Eq, PartialEq)]
+pub enum ProofGenerationError {
+    #[error("not enough funds in account")]
+    NotEnoughFunds,
+    #[error("transfer fee calculation error")]
+    FeeCalculation,
```

Comment:
Should these errors be changed or removed?

Reply:
Oh great point! I removed it. Thanks!

Reply:
Ok cool! Can you also get rid of `NotEnoughFunds`?

Reply:
Oh oops, removed. Thanks!